doc/notebooks/XarrayforArviZ.ipynb

```diff
@@ -13,21 +13,33 @@
    "source": [
     "While ArviZ supports plotting from familiar datatypes, such as dictionaries and numpy arrays, there are a couple data structures central to ArviZ that are useful to know when using the library. \n",
     "\n",
-    "They are\n",
-    "* xarray.Dataset\n",
-    "* arviz.InferenceData\n",
-    "* NetCDF\n",
+    "They are  \n",
+    "\n",
+    "<ul class=nbtable>\n",
+    "  <!-- HTML Table to allow proper rendering in sphinx -->\n",
+    "  <li>xarray.Dataset</li>\n",
+    "  <li>arviz.InferenceData</li>\n",
+    "  <li>netCDF</li>\n",
+    "</ul> \n",
```

Comment:
Should these be hidden? If I check these with view, I can see them

    <ul ...
    </ul>

Maybe if they are translated to HTML they are hidden

Reply:
Aw shoot that's annoying. They seem to get hidden when Sphinx renders them, but not github apparently. I'll make another fix